vaccine_feed_ingest/runners/nv/immunizenevada_org/normalize.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest.schema import schema
+from vaccine_feed_ingest.utils.normalize import provider_id_from_name
+
+
+def _get_id(site: dict) -> str:
+    return f"immunizenevada_org:{site['id']}"
+
+
+def _get_address(address: str) -> schema.Address:
+    parts = address.split(",")
+    street2 = None
+    if len(parts) == 5:
+        street2 = parts[1]
+
+    result = schema.Address(
+        street1=parts[0],
+        street2=street2,
+        city=parts[-3],
+        state=parts[-2],
+        zip=parts[-1],
+    )
+    return result
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = None
+
+    phone = None
+    if "contact-phone" in site:
+        phone = site["contact-phone"]
```

Comment:
It looks like some phone numbers in the source are formatted, and some are not. Could you make sure that all normalized phone numbers end up like `(###) ###-####`?